pom.xml

```diff
@@ -111,6 +112,11 @@
     </dependency>
     <dependency>
       <groupId>org.pac4j</groupId>
+      <artifactId>j2e-pac4j</artifactId>
+      <version>${j2e-pac4j.version}</version>
```

Comment:
I'm wondering: shouldn't we make the `jax-rs-pac4j` and `j2e-pac4j` dependencies `optional` to let users pull them in explicitly? Or is it too tricky?